CHANGELOG.md

```diff
@@ -25,6 +27,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bumped `pypdf` from version 4.2.0 to 6.0.0 in `docs/environments/gcpy_environment_py312.yml`
 - Bumped `pypdf` from version 5.3.1 to 6.0.0 in `docs/environments/gcpy_environment_py313.yml`
 - Moved `Ap` and `Bp` parameters that define vertical grids from `grid.py` to `vgrid_defs.py`
+- Replaced `[-180, 180, -90, 90]` with `GLOBAL_LL_EXTENT` in functions
+- Replaced `[1, 170, -90]` with `NO_STRETCH_SG_PARAMS` in functions
+- Renamed `skip_these_vars` to `SKIP_THESE_VARS` to conform to Python style for constants
+- Modified code in `grid.py`, `regrid.py`, `file_regrid.py`, and `cstools.py`, and `raveller_1D.py`  as directed by Pylint to better conform to the Python style guide
+- Renamed the following routines in `gcpy/grid.py` to conform to snake-case naming convention
+  - `csgrid_GMAO      --> csgrid_gmao`
```

Comment:
I think this level of detail in the changelog is a bit too much. Best keep it simple, e.g. "Renamed grid routines to use all lower-case to conform to snake-case naming convention". Same comment for other sections of changelog that have bulleted lists.

Reply:
Thanks @lizziel, now updated